ui-kit/src/main/res/values/styles_fonts.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="TextAppearance">
+        <item name="android:textColor">@color/night</item>
+    </style>
+
+    <!--region TextAppearance.Regular-->
+    <style name="TextAppearance.Regular">
```

Comment:
I would rather name them according to Figma names. it would be easier for us to navigate and understand which style to use

Default / Regular / LargeTitle -> Title/Large Title/Regular
Default / Regular / Title3 -> Title/H3/Regular

or force designers to rename their styles

<img width="1032" alt="Screen Shot 2022-06-09 at 1 56 11 pm" src="https://user-images.githubusercontent.com/100791035/172831262-0409ea9b-3b68-47b5-80c5-d958898c2bef.png">
es. 

it would be better for us

Reply:
they will rename H3 to normal readable Title3
i already asked them about it